src/digital_identity_service/index.js

```diff
@@ -260,11 +260,11 @@ class DigitalIdentityService {
   /**
    * @param {string} receiptId
```

Comment:
missing jsDoc `@returns {Promise<GetShareReceiptResult>}`

Reply:
Missing `@returns` for the methods fetchReceipt and fetchReceiptItemKey as well.

Reply:
👍